pop-api/src/v0/sponsorships/events.rs

```diff
@@ -0,0 +1,23 @@
+use super::*;
+
+/// Event emitted when a new sponsorship is registered.
+#[ink::event]
+pub struct NewSponsorship {
```

Comment:
Why do we need these? They only exist in the fungibles instance to allow contract devs to reused PSP-standards compliant event structures. Can remove imo.

Reply:
Introduced them at first to make sure the api integration was working properly, but indeed it makes no sense to have repeated events when they are already being emitted from the runtime.

They should be removed now